tests/cloner_test.go

```diff
@@ -716,10 +721,14 @@ func doFileBasedCloneTest(f *framework.Framework, srcPVCDef *v1.PersistentVolume
 
 	fmt.Fprintf(GinkgoWriter, "INFO: wait for PVC claim phase: %s\n", targetPvc.Name)
 	utils.WaitForPersistentVolumeClaimPhase(f.K8sClient, targetNs.Name, v1.ClaimBound, targetPvc.Name)
-	completeClone(f, targetNs, targetPvc, filepath.Join(testBaseDir, testFile), fillDataFSMD5sum)
+	sourcePvcDiskGroup, err := f.GetDiskGroup(f.Namespace, srcPVCDef)
+	fmt.Fprintf(GinkgoWriter, "INFO: %s\n", sourcePvcDiskGroup)
+	Expect(err).ToNot(HaveOccurred())
+
+	completeClone(f, targetNs, targetPvc, filepath.Join(testBaseDir, testFile), fillDataFSMD5sum, sourcePvcDiskGroup)
 }
 
-func completeClone(f *framework.Framework, targetNs *v1.Namespace, targetPvc *v1.PersistentVolumeClaim, filePath, expectedMD5 string) {
+func completeClone(f *framework.Framework, targetNs *v1.Namespace, targetPvc *v1.PersistentVolumeClaim, filePath, expectedMD5, sourcePvcDiskGroup string) {
```

Comment:
I might be missing it, but is there a test that imports a disk image, then clones it (including the fsgroup)?

Reply:
No but we haven't modified the clone behaviour in this PR.

Reply:
I understand, but to me, the goal of this PR is to ensure that all disk images have the appropriate fs group.  Cloning from an import seems like a pretty common use case.  Should we not confirm that it works correctly?

Reply:
There is no specific test, but at the end of completeClone, I compare the fsGroup of the clone against the fsGroup of the source. And that is called in a bunch of the clone tests.